lib/app_store/connect.rb

```diff
@@ -205,7 +207,9 @@ def create_review_submission(build_number:, version: nil)
       execute do
         build = get_build(build_number)
 
-        edit_version = app.get_edit_app_store_version(includes: "build")
+        edit_version = app
+          .get_app_store_versions(includes: "build", filter: INFLIGHT_RELEASE_FILTERS)
+          .find { |v| v.build&.version == build_number.to_s }
```

Comment:
I feel like this is repeated a lot, can we extract?

Reply:
this is actually not repeated.